tensorflow/python/tools/saved_model_cli.py

```diff
@@ -885,6 +885,9 @@ def _create_example_string(example_dict):
     if not isinstance(feature_list, list):
       raise ValueError('feature value must be a list, but %s: "%s" is %s' %
                        (feature_name, feature_list, type(feature_list)))
+    # Boş feature listesi anlaşılır bir hata ile reddedilsin.
```

Comment:
Again, please at least remove the comments that the AI generates when they are useless.

If you continue sending AI slop we might close the PRs.